.github/workflows/integration-tests.yml

```diff
@@ -43,6 +49,8 @@ jobs:
         run: |
           uv sync --extra dev --extra test
           uv pip install ollama faiss-cpu
+          # always test against the latest version of the client
+          uv pip install git+https://github.com/meta-llama/llama-stack-client-python.git@main
```

Comment:
this will eventually bite us because it assumes a particular order of component updates; an overlapping compatibility ranges would be preferable, but it would require different engineering practices for API changes than what's practiced here at the moment.

(We could also have a monorepo with client bindings managed in the main repo; in which case both could be updated - and validated - at the same time. This would also require a change in practices.)